src/OpenTelemetry.Instrumentation.ConfluentKafka/ConsumeAndProcessMessageHandler.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+using Confluent.Kafka;
+
+namespace OpenTelemetry.Instrumentation.ConfluentKafka;
```

Comment:
@g7ed6e Should this also go into `Confluent.Kafka` namespace since it is only used with the extension? Thinking it would save callers from having to add a `using` just for this type. If you decide to do that, maybe change the name `ConsumeAndProcessMessageHandler` -> `OpenTelemetryConsumeAndProcessMessageHandler` just to avoid any potential collisions?

Reply:
@CodeBlanch It will ease discoverability indeed. done!